### ISSUE TITLE
Support `arm64` architectures

### ISSUE THREAD

**Issue by jpmcb** [ublue-os/bluefin#755]
### Describe the bug

Running the docker container from an arm based machine:

```
❯ docker run -it ghcr.io/ublue-os/bluefin:latest bash
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
bash-5.2#
```

pulls the image successfully and will mount/run my command, but it throws a warning about the architecture. 

And it looks like there is only 1 x86 manifest:

```
❯ crane manifest ghcr.io/ublue-os/bluefin:latest | jq
{
  "schemaVersion": 2,
  "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
  "config": {
    "mediaType": "application/vnd.docker.container.image.v1+json",
    "size": 23813,
    "digest": "sha256:1169c58f27ec705ec18099dfd6105215ba33744cab7a43aff9eefbcfff992c57"
  },
  "layers": [
        ...
    ]
}
```

---

I'm also not seeing arm supported ISOs in the releases. And from reading the `.github/workflow` scripts, I'm not seeing builds against arm64 so my assumption is it's not supported.

### What did you expect to happen?

It'd be great if bluefin supported arm architectures: this would open the door for alot of fun side projects on raspberries pies, edge IOT devices, etc.

### Output of `rpm-ostree status`

```shell
n/a
```

### Extra information or context

FYI, here is the mac I ran this from:

```
OS: macOS 13.3 22E252 arm64
CPU: Apple M2 Max (arm64)
```

**Comment by castrojo** [comment]
Yeah we don't have arm builds yet: https://github.com/ublue-os/main/issues/365

**Comment by castrojo** [comment]
We'll inherit this from main so closing here.

**Comment by castrojo** [comment]
Ok akmods are being enabled. I think we should kill off hwe before we enable this so we don't saturate the builders, since Aurora and Bazzite will also be turning this on. And we have to turn on F43 builders soon. I don't think we should wait for release, when these build let's just announce them. 

How's this:

- Nvidia: No -> aarch/nvidia is ML territory -> Bluefin GDX
- GTS: No -> Bluefin LTS - let's just put them on one side or the other, not the middle
- stable: yes -> this should be the default for ARM users
- latest: yes -> must provide the same version as Fedora

Then we'll also turn off nvidia-closed in October, so we should be good!

**Comment by tulilirockz** [comment]
We either need `ublue-os/main` to have aarch64 or bluefin itself needs to not use main so that we can get this. Everything else should be just fine. Would be awesome to get this though!

**Comment by shvetsnikita** [comment]
Just wondering, what was idea [here](https://github.com/ublue-os/main/pull/524#issuecomment-2781474525)? I tried to find it, but probably missed it somewhere.
